utils/__init__.py

```
@@ -0,0 +1,36 @@
+from .manager import PathManager, LogManager, ConfigManager, ProgressorManager
+from .shared import *
```

Ruff (0.11.9)

`from .shared import *` used; unable to detect undefined names

(F403)